warp-academy-docs/standards/atomic-asset.md

```diff
@@ -0,0 +1,121 @@
+# Introduction
+
+## What is atomic asset?
+
+An atomic asset is the concept, where **asset** (like an image) and **smart contract** are deployed atomically - in one logical transaction.
+It is possible, thanks to unique properties of arweave blockchain. In a typical blockchain asset is saved on some storage (for example IPFS) and only smart contract is deployed on-chain (for example ERC721). Why to choose atomic asset:
+
+- Standards like ERC-721 on Ethereum usually point to the external links of the NFT’s asset, and are stored outside of the Ethereum Virtual Machine, thus you have no guarantee that it will be stored there permanently. In scenario where storage provider is centralized you could wake up with empty NFT (pointing to non-existing URI) :( However even if u are using decentralized storage like IPFS, it doesn't provide strong guarantees about data availability like Arweave do. (you can read more [here](https://arweave.news/nfts-storage-arweave-vs-ipfs/)). In one example an artist named [neitherconfirm](https://twitter.com/neitherconfirm/status/1369285946198396928?s=20) changed the media files referenced by 26 of his NFTs to stock images of rugs. The artist later commented, “All discussions about the value of NFTs are meaningless as long as the token is not inseparable from the artwork its
+- With atomic assets, there is no possibility to create two (or more) contracts pointing to the same data, definition of the ownership is bounded to the asset not the other way around. "With Atomic NFTs, the address of the NFT smart contract _is_ the address of the NFT’s data. Permanent, on-chain assets, and no external metadata. One address. How NFTs were supposed to be" - Sam Williams, Arweave founder.
+- Take the red pill 💊! Be the owner of your data, not the link to the data! 😎
+
+## What is atomic-asset standard?
+
+As we described earlier Atomic asset is a general idea, like [NFT](https://en.wikipedia.org/wiki/Non-fungible_token), this kind of definition is great, however, not specific enough for composability (example: Let's imagine, that you want to build `atomic asset` marketplace, you have to know how to consume existing `atomic assets` - which method to call on contract to transfer it and with which params).
+**atomic-asset** standard is here to help, by standardizing the interface describing the smart contract behind `atomic-asset`. So when building new marketplace (or other product), you don't have to think about multiple implementations, per project.
```

Comment:
**atomic-asset** standard is here to help, by determining the interface describing the smart contract behind `atomic-asset`. So when building a new marketplace (or other any product), you don't have to think about multiple implementations, per project.